x/auth/client/cli/tx_sign.go

```diff
@@ -114,22 +109,23 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 				if err != nil {
 					return fmt.Errorf("error getting account from keybase: %w", err)
 				}
-				err = authclient.SignTx(txFactory, clientCtx, fromName, txBuilder, true)
+				err = authclient.SignTx(txFactory, clientCtx, fromName, txBuilder, true, true)
```

Comment:
Default behavior in the `tx sign` command is "append=true". Why is this not the default when doing sign batch?

If we don't have the `--append` flag as an option in the `sign batch` command, I still think we should probably keep the same default behavior as documented in `tx sign`.

Reply:
There is no `overwrite` nor `append` option in the `sign-batch` command. The original behavior was to not append. 
Do you think we should change this as well?

Reply:
It's my understanding that the original behavior was intended to be the same as the intended original behavior for `tx sign` (append by default). Pretty sure that @alessio wrote thie `sign batch` command, so i'll default to whatever he things the intended behavior should be.

Reply:
OK, I will check with him tomorrow. Now it behaves as it was before (overwrite=true is the original behavior of `SignTx`)

Reply:
Yes, @clevinson is right. This looks good :+1:

Reply:
I think I've misread this and I've realized only now that this is about the `sign-batc` command - sorry about that. 

> It's my understanding that the original behavior was intended to be the same as the intended original behavior for tx sign (append by default)

Cory is right. So IMHO the original default behaviour should still be the default one. I think this implies `--overwrite=false`, maybe I'm wrong?

Reply:
this is exactly what we have (`--overwrite=false` by default)